src/components/Myprofile.js

```diff
@@ -0,0 +1,40 @@
+import { useSelector } from 'react-redux';
+import { ListGroup } from 'react-bootstrap';
+/* import './Myprofile.css'; */
+
```

Comment:
- [ ] Please remove commented code as it makes your work look untidy 👍

Reply:
@codepantha  done